gradle/docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  strongbox-web-integration-tests-nuget:
+    image: strongboxci/alpine:jdk8-gradle-4.5
+    volumes:
+      - $HOME/.m2/repository:/home/jenkins/.m2/repository
+      - ./:/workspace
+    command: /bin/bash -c "cd /workspace && mvn clean install"
```

There is no need to do a `cd` inside the command. You can use `working_dir` which will do it for you.
```suggestion
    command: mvn clean install
    working_dir: /workspace
```